apps/server/src/seed.ts

```diff
@@ -1,3 +1,3 @@
- import seedDb from "./lib/mongoose/seeders/seeder";
```

Comment:
`yarn seed` will run `seeder-aroldev` unless you change it back to `seeder-random`